tokens/cnft-vault/app/readAPI.ts

```diff
@@ -0,0 +1,48 @@
+// I recommend using a WrappedConnection for production
+// as it supports more readAPI functionality
+// this is just a subset of functions for quick availabiity
+
+import axios from "axios";
+
+// TODO change to your API key (this is mine on free-tier)
+const HELIUS_RPC = "https://rpc-devnet.helius.xyz/?api-key=30536abf-e8e7-444f-a255-18e9a0c27e8b";

```

Comment:
Maybe better point to metaplex read api? Just in case at some point you upgrade your tier  or invalidate your api etc.

Reply:
Agreed & you don't want your endpoint in here, even if it is free tier